gears/gears-compose/build.gradle.kts

```
@@ -0,0 +1,27 @@
+plugins {
+    convention.library.android
+}
+
+version = "1.0.0"
```

Let's start from 0.1.0

```suggestion
version = "0.1.0"
```